src/Symfony/Installer/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony Installer package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Installer\Tests;
```

Comment:
I suggest putting the tests in ``tests/`` at the root rather than in ``src/``. It will make it much easier to exclude the testsuite from the phar.
In Symfony itself, we have the tests in the codebase because we want to bring them in subtree splits, and so it was necessary to have them in the same subtree. And we have them in a subfolder of the code in bundles because of the convention for bundles being created in the pre-PSR-4 times. It is now possible to create repos for bundles with tests and source separated (Symfony does not actually care about it, as it will never look for the tests anyway)